### PR TITLE
Add tests specific for WRS

### PR DESCRIPTION
**Pull request by liangxia**
Context, https://redhat-internal.slack.com/archives/GH9JHQFKN/p1710981765784209, and test [log](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/job-history/gs/qe-private-deck/pr-logs/directory/rehearse-50954-periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetalds-ipi-ovn-ipv4-fips-f14-wrs)

### CODE REVIEW

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-baremetalds-ipi-ovn-ipv4-fips-f14-wrs

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetalds-ipi-ovn-ipv4-fips-f14-wrs

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetalds-ipi-ovn-ipv4-fips-f14-wrs

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetalds-ipi-ovn-ipv4-fips-f14-wrs

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -1687,6 +1687,22 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-ui
     workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn
+- as: baremetalds-ipi-ovn-ipv4-fips-f14-wrs
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1690]:
```suggestion
- as: baremetalds-ipi-ovn-ipv4-fips-wrs-f14
```

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:1690]: I think `wrs` is more like a new test type. It's not associated to the workflow/profile names.

**Comment by liangxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-baremetalds-ipi-ovn-ipv4-fips-f14-wrs

**Pull request merged**